### ISSUE TITLE
Adding in annotated image of the point source selector

### ISSUE THREAD

**Issue by mschwamb** [fraserw/trippy#4]
I found it a little hard to follow your description of the point source selector. I made an annotated figure that I think would be helpful to what you have already written

<img width="829" alt="screen shot 2016-01-15 at 10 56 43 am" src="https://cloud.githubusercontent.com/assets/753325/12344408/9a2277e8-bb77-11e5-844c-83c677126933.png">

**Comment by fraserw** [comment]
There is no good way to display pdfs inside markdown, and adding code to load this figure would only confuse things. There is no way to do what you ask sadly.

**Comment by mschwamb** [comment]
There's a may to add images in markdown. 

```
    ![Alt text](/path/to/img.jpg)

    ![Alt text](/path/to/img.jpg "Optional title")
```

But absolutely your choice if you want to add it or not.

**Comment by fraserw** [comment]
There be the catch. What the hell is the path of a ipython notebook cached in someone’s browser? lol

**Comment by fraserw** [comment]
Just going through the old issues, and this one never seemed to get sorted out. Do you know the way to incorporate this image with the correct path?

**Comment by Mikea1985** [comment]
Could we use something like

    if tutorial==True:
        pyl.text(x1, y1, 'Histogram of FWHM', color='r')
        pyl.text(x2, y2, 'Histogram of fit chi values', color='r')
        etc.

Which would make the text appear directly on the figure, but only when a tutorial flag is set to True (would need to add a tutorial keyword to the psfStarSelector call, but it could be optional with False as default). 
If desirable, I could have a look at implementing that.
